staramr/blast/results/plasmidfinder/BlastResultsParserPlasmidfinder.py

```diff
@@ -0,0 +1,59 @@
+from os import path
+
+from staramr.blast.results.BlastResultsParser import BlastResultsParser
+from staramr.blast.results.plasmidfinder.PlasmidfinderHitHSP import PlasmidfinderHitHSP
+
+"""
+Class used to parse out BLAST results for Plasmidfinder.
+"""
+
+
+class BlastResultsParserPlasmidfinder(BlastResultsParser):
```

Comment:
I think for the directory containing these classes `staramr/blast/results/plasmidfinder/` you need to create a file `__init__.py` to indicate it's a package. I believe this is the source of error for the failing tests right now.

Reply:
Resolved in d36d011